components/common/accordion/message_item.tsx

```diff
@@ -10,7 +10,7 @@ type Props = {
     severity: 'info' | 'warning' | 'error';
 }
 
-const MessageItem: React.FC<Props> = ({title, description}: Props): JSX.Element | null => {
+const MessageItem = ({title, description}: Props): JSX.Element | null => {
```

Comment:
just clean up for the effort of removing the unnecessary React.FC type from functional/hook components